### ISSUE TITLE
Build issues

### ISSUE THREAD

**Issue by giilby** [kadencewp/kadence-blocks#216]
Hello! I'd love to contribute some bug-fixes and new features to `kadence-blocks`, but I'm having trouble with the build system added by #190. When I run `gulp build`, nothing is output to the `dist` folder: instead, I only see some CSS and JS assets output within `includes/assets`. If I run `gulp webpack`, I get a lot of output to `dist`, but the plugin doesn't work correctly, and I also see that it wipes out the `dist/assets` folder, which seems to be required for compiled CSS. Lastly, I simply cannot get `gulp watch` to work correctly.

The only way I've been able to test my changes thus far is to run `npm run build`, then `gulp build`, then copy `includes/assets` into the `dist` folder manually. After that, the blocks run well on my test site.

It's very possibly that I simply don't understand which commands to run when, in which case I'd just love some pointers. Thanks!

**Comment by oakesjosh** [comment]
Awesome!

We've reverted some of the changes from #190 and are now mostly using webpack. You're running the right commands. You should just have to run `gulp build` once, then run `npm run build` as needed or `npm run watch`

Just a note: The master branch contains a **lot** of refactoring and other changes that haven't been released yet. Some of the changes are listed in the readme under `2.5.0  Beta version`, but every block has been modified and you may come across some bugs.

**Comment by giilby** [comment]
Good to know...thanks for the heads-up. Is there a "stable" commit that I should work from? Also, how does this repo match up with https://wordpress.org/plugins/kadence-blocks/? The latest version listed on wordpress.org is 2.4.13 (released on July 29th), but I don't see any reference to that version in this repository.

**Comment by oakesjosh** [comment]
The latest is probably most stable. This repo is currently missing the 2.4.9 - 2.4.13 minor releases that were pushed to .org

**Comment by giilby** [comment]
Will this repo eventually receive those changes? Or does the official plugin have a completely separate repo?

**Comment by oakesjosh** [comment]
Yes it will. This is the official repo. Those releases will probably get merged in within the next couple weeks
